domain/secretbackend/state/state.go

```diff
@@ -288,10 +288,6 @@ ORDER BY b.name`, SecretBackendRow{})
 }
 
 func (s *State) getSecretBackend(ctx context.Context, tx *sqlair.TX, columName string, v string) (*secretbackend.SecretBackend, error) {
-	db, err := s.DB()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
 	q := fmt.Sprintf(`
 SELECT
     b.uuid                  AS &SecretBackendRow.uuid,
```

Comment:
we can use * here right?

Reply:
Ah, because this join outcome has two name fields we require.
So we need the AS for aliasing.